tasks/3.html

```diff
@@ -12,7 +12,16 @@ <h1 class="global-heading">&nbsp;</h1>
 
         <main class="global-results">
             <style>
-                /* your styles here */
+                div
+                {
+                    border: 2px solid #000;
+                }
+
+                article.global-scope.global-scope_3 > div > div
```

Comment:
Можно ли здесь обойтись без всей первой части селектора?
```suggestion
                div div
```